utils/getScenarioContent.tsx

```diff
@@ -0,0 +1,98 @@
+import { Claim, ClaimDetailsContent, ClaimStatusContent, ScenarioContent } from '../types/common'
+
+export enum ScenarioType {
+  PendingDetermination = 'Pending determination scenario',
+  BasePending = 'Base state with pending weeks',
+  BaseNoPending = 'Base state with no pending weeks',
+}
+
+/**
+ * Identify the correct scenario to display.
+ *
+ * @param {Qbject} claim
+ * @returns {Object}
+ */
+export function getScenario(claimData: Claim): ScenarioType {
+  // The pending determination scenario: if claimData contains any pendingDetermination
+  // objects
+  // @TODO: refactor with more detailed pending determination scenarios #252
+  if (claimData.pendingDetermination && claimData.pendingDetermination.length > 0) {
```

Comment:
If `pendingDetermination` is `false` and we use `if (claimData.pendingDetermination?.length > 0)`, that should return `false` right?

Reply:
it would not - `.?` only handles nullish (`null` or `undefined`) so if we think it could come back as `false` we would want to leave the longer version

Reply:
Ah I see. Then I think we should leave the longer version.